sorc/emcsfc_snow2mdl.fd/model_grid.f

```diff
@@ -58,57 +58,53 @@ module model_grid
  real, allocatable              :: lsmask_mdl_sav (:,:)
  real                           :: resol_mdl  ! in km
 
- contains
+!> 
+!! @brief read latitude, longitude, land/sea mask on the    model grid.
+!!
+!! @author George Gayno org: w/np2 @date 2005-dec-16
+!!              
+!!
+!! program history log:
+!! 2005-dec-16  gayno    - initial version
+!! 2007-nov-30  gayno    - improved method for thinning gfs grids
+!!                         added nam b-grids
+!! 2014-sep-29  gayno    - add option to read lat,lon and mask
+!!                         data in grib2.
+!!
+!! usage: call read_mdl_grid_info
+!!
+!!   input argument list:  n/a
```

Comment:
Remove any documentation element that says n/a or none. It is not conveying information.